src/components/bid_card/BidCard.js

```diff
@@ -3,14 +3,26 @@ import { dateFormatter } from 'utilities/date';
 import { convertPenniesToDollars } from 'utilities/price';
 import './BidCard.scss';
 
-const BidCard = ({ timestamp, bid_price, userName, white }) => (
-  <section className={`bid--container ${white && 'bid--white'}`}>
-    <div className='bid--container-info'>
-      <h2>{userName} Bid</h2>
-      <h5>{dateFormatter(timestamp)}</h5>
-    </div>
-    <h1 className='bid--container-price'>${convertPenniesToDollars(bid_price)}</h1>
-  </section>
-)
+const BidCard = ({ timestamp, bid_price, userName, white, winner }) => {
+  const bidClass = () => {
+    if (white && !winner) {
+      return 'bid--white';
+    } else if (winner) {
+      return 'bid--winner';
+    }
+  }
+
+  const bidMessage = winner ? "Won" : "Bid";
+
+  return (
+    <section className={`bid--container ${bidClass()}`}>
+      <div className='bid--container-info'>
+        <h2>{userName} {bidMessage}</h2>
+        <h5>{dateFormatter(timestamp)}</h5>
```

Comment:
nitpicky, but this should probably be called `formatDate`. I like naming functions as verbs - dateFormatter seems more like the name of an object